target/linux/ramips/dts/mt7620n_ravpower_rp-wd03.dts

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This (and/or the `dtsi`) is a re-license, IMHO.

You have to ask all the contributors do they agree with the change!

Reply:
I have no issue removing this, to keep it as it was before - but I was told that this was missed in some cases before (an error, it happens), and now it has to be included. So added it based on that.

But I get your point, no issue. Let me add the folks who it seems contributed to this file before (if I have it right!). Any concerns with this addition ... @adschm, @mans0n, @mwarning, @981213? 

Thanks!

Reply:
@mwarning aye (although, I do not remember contributing here..)

Reply:
I thought all files in the folder need license identifier, but DTSI does not have DTS tag, so DTS has both, DTSI only has license

Reply:
Grep for `Signed-off-by:` in the commit history!

Here is how GitHub sees them:
* [history](https://github.com/openwrt/openwrt/commits/d0f295837a03f7f52000ae6d395827bdde7996a4/target/linux/ramips/dts/mt7620n_ravpower_wd03.dts) of `mt7620n_ravpower_wd03.dts` up to commit d0f295837a03f7f52000ae6d395827bdde7996a4
* [history](https://github.com/openwrt/openwrt/commits/dd3ec40d3d07f9fb419936b27f386217a0c68fb0/target/linux/ramips/dts/WD03.dts) of `WD03.dts`

Reply:
@arrmo , yup, please keep the header and do the re-license work!

Reply:
> But I get your point, no issue. Let me add the folks who it seems contributed to this file before (if I have it right!). Any concerns with this addition ... @adschm, @mans0n, @mwarning, @981213?

Acked-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>

Reply:
Sorry, to understand - "re-license" work. Do you mean to get everyone on the lists above to approve this? Easier would be to remove it, go back the way it was 🤣.